src/main/kotlin/com/criptext/mail/scenes/mailbox/data/SendMailWorker.kt

```diff
@@ -176,6 +176,8 @@ class SendMailWorker(private val signalClient: SignalClient,
                     finalAttachments.add(file)
                 }
             }
+        }else if(attachments.isNotEmpty()){
+            finalAttachments.addAll(attachments)
```

Comment:
What happens If I have a forwarded email with attachments and I add more attachments? This ones won't be included in the email.